tests/test_ade20k2020_format.py

```diff
@@ -0,0 +1,68 @@
+from datumaro.components.extractor import AnnotationType, LabelCategories, Mask
+from datumaro.util.test_utils import compare_datasets
+
+from tests.requirements import Requirements, mark_requirement
+
+DUMMY_DATASET_DIR = osp.join(osp.dirname(__file__), 'assets', 'ade20k2020_dataset')
+
+class Ade20k2020ImporterTest(TestCase):
+    @mark_requirement(Requirements.DATUM_399)
+    def test_can_import(self):
+        expected_dataset = Dataset.from_iterable(
+            [
+                DatasetItem(id='1', subset='training',
+                    image=np.ones((5, 5, 3)),
+                    annotations=[
+                        Mask(image=np.array([[0, 0, 1, 1, 1]] * 5), label=0,
+                            group=401, z_order=0, id=401),
+                        Mask(image=np.array([[0, 1, 0, 0, 0]] * 5), label=1,
+                            group=1831, z_order=0, id=1831),
+                        Mask(image=np.array([[0, 0, 0, 1, 1]] * 5), label=2,
+                            id=774, group=774, z_order=1),
+                        Mask(image=np.array([[0, 0, 1, 1, 1]] * 5), label=0,
+                            group=0, z_order=0, attributes={'part_level': 0}),
+                        Mask(image=np.array([[0, 1, 0, 0, 0]] * 5), label=1,
+                            group=1, z_order=0,
+                            attributes={'walkin': True, 'part_level': 0}),
+                        Mask(image=np.array([[0, 0, 0, 1, 1]] * 5), label=2,
+                            group=2, z_order=1, attributes={'part_level': 1}),
+                    ]),
+                DatasetItem(id='2', subset='validation',
+                    image=np.ones((5, 5, 3)),
+                    annotations=[
+                        Mask(image=np.array([[0, 0, 1, 1, 1]] * 5), label=0,
+                            group=401, z_order=0, id=401),
+                        Mask(image=np.array([[0, 1, 0, 0, 0]] * 5), label=1,
+                            group=1831, z_order=0, id=1831),
+                        Mask(image=np.array([[0, 0, 0, 0, 1]] * 5), label=3,
+                            group=2122, z_order=2, id=2122),
+                        Mask(image=np.array([[0, 0, 0, 1, 1]] * 5), label=2,
+                            group=774, z_order=1, id=774),
+                        Mask(image=np.array([[0, 0, 1, 1, 1]] * 5), label=0,
+                            group=0, z_order=0, attributes={'part_level': 0}),
+                        Mask(image=np.array([[0, 1, 0, 0, 0]] * 5), label=1,
+                            group=1, z_order=0,
+                            attributes={'walkin': True, 'part_level': 0}),
+                        Mask(image=np.array([[0, 0, 0, 1, 1]] * 5), label=2,
+                            group=2, z_order=1, attributes={'part_level': 1}),
+                        Mask(image=np.array([[0, 0, 0, 0, 1]] * 5), label=3,
+                            group=3, z_order=2, attributes={'part_level': 2}),
```

Comment:
Why some masks do not have `id` and a leading group member, if they are grouped?

Reply:
Masks with `id` are used for semantic segmentation, where id is assigned a class index

Reply:
Mask id should be an instance id, if specified. What is the class index?

Reply:
It is index of class that encode for each pixel in the RGB image:
https://github.com/openvinotoolkit/datumaro/blob/bbc28fd4bf2ebd21ab39cdfede663a42ec53f1b0/datumaro/plugins/ade20k2020_format.py#L144-L148
 (a similar structure exists in the COCO Panoptic format)

Reply:
I'm still not convinced about the missing id field. It must represent instance id on the leading annotation in the group.